tt_metal/jit_build/settings.hpp

```diff
@@ -26,6 +26,7 @@ class JitBuildOptions {
     // We can keep for future WH support, otherwise not used in GS
     bool fp32_dest_acc_en;
     std::vector<UnpackToDestMode> unpack_to_dest_mode;
+    bool bfp8_pack_precise;
```

Comment:
To me this looks like an uninitialized member variable. 
See 
https://github.com/tenstorrent/tt-metal/blob/main/best_practices.md#15-initialize-primitive-types-on-declaration

Reply:
added initial value to settings.cpp like for `fp32_dest_acc_en`